src/raven/client.clj

```diff
@@ -286,23 +276,17 @@
 
 (defn perform-http-request
   [context dsn ts payload]
-  (let [json-payload  (json/generate-string payload)
+  (let [json-payload                 (json/write-value-as-string payload)
         {:keys [key secret uri pid]} (parse-dsn dsn)
-        sig                          (sign json-payload ts key secret)
-        p (promise)]
+        sig                          (sign json-payload ts key secret)]
     ;; This is async, but we don't wait for the result since we don't really care if the
     ;; event makes it to the sentry server or not (we certainly don't want to block until
     ;; it fails).
-    (http/async-request (get-http-client context)
-                  {:uri            (format "%s/api/store/" uri)
-                   :request-method :post
-                   :headers        {"X-Sentry-Auth" (auth-header ts key sig)
-                                    "User-Agent"    user-agent
-                                    "Content-Type"  "application/json"
-                                    "Content-Length" (count json-payload)}
-                   :transform      st/transform
-                   :body           json-payload}
-                  (constantly nil)))) ;; Noop
+    (http/post (format "%s/api/store/" uri)
+               {:headers {"X-Sentry-Auth"  (auth-header ts key sig)
+                          "Content-Type"   "application/json; charset=utf-8"
```

Comment:
`charset=utf-8` is overkill for `application/json`.

I guess, aleph does the `content-length` for us too. https://github.com/ztellman/aleph/blob/651ea01304166652d38d1f09936f73ccb062bbf8/src/aleph/http/core.clj#L233-L235